terraform/modules/psn/main.tf

```diff
@@ -1,83 +1,84 @@
 variable "vpc_id" {
-  type        = "string"
+  type        = string
   description = "The ID of the VPC in which the endpoint will be used."
 }
 
 variable "vpc_endpoint" {
-  type        = "string"
+  type        = string
   description = "The service name, in the form com.amazonaws.region.service for AWS services."
 }
 
 variable "subnet_ids" {
-  type        = "list"
+  type        = list(string)
   description = "The ID of one or more subnets in which to create a network interface for the endpoint."
 }
 
 variable "security_group_id" {
-  type        = "string"
+  type        = string
   description = "The security group to allow access to the PSN VPC Endpoint."
 }
 
 resource "aws_vpc_endpoint" "psn_service" {
-  vpc_id            = "${var.vpc_id}"
-  service_name      = "${var.vpc_endpoint}"
+  vpc_id            = var.vpc_id
+  service_name      = var.vpc_endpoint
   vpc_endpoint_type = "Interface"
 
-  security_group_ids = ["${aws_security_group.psn_endpoint.id}"]
+  security_group_ids = [aws_security_group.psn_endpoint.id]
 
-  subnet_ids          = ["${var.subnet_ids}"]
+  subnet_ids          = var.subnet_ids
```

Comment:
```suggestion
  subnet_ids          = [var.subnet_ids]
```
no?

Reply:
No, subnet_ids is already a list, we don't want to turn it into a list of lists.